### PR TITLE
Add `TWC 2023` wiki article

### PR DESCRIPTION
**Pull request by LeoFLT**
🥁
PRing this ahead of time to split it from the (upcoming) news article PR.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/TWC/2023/en.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+6. The [Hidden](/wiki/Game_modifier/Hidden), [Hard Rock](/wiki/Game_modifier/Hard_Rock) and [Double Time](/wiki/Game_modifier/Double_Time) brackets will be played with the respective mods activated along with the No Fail mod.
+7. The [FreeMod](/wiki/Game_modifier#freemod) bracket will have Free Mods activated. The valid mod combinations for each player are:
+   - [NoMod](/wiki/Game_modifier#nomod) – no mods selected
+   - Hidden
+   - Hard Rock
+   - Hidden + Hard Rock
+8. For FreeMod, each team is **required** to have both Hidden AND Hard Rock selected among its players. A player may shoulder all the mods for their team (e.g. one player selects Hidden + Hard Rock, and the remaining players select no mods (NoMod)), or it may be split among all members (e.g. the first player selects Hidden, the second selects Hard Rock, and the third selects no mods (No Mod)). Teams may "overmod", that is, select more mods than what is required.
+9. Failed scores will be counted in their integrity. To that end, the [No Fail](/wiki/Game_modifier/No_Fail) mod will be enforced on every beatmap.
+   - Teams that purposefully disregard this instruction and team fail as a result will lose said game.
```
Comment (Walavouchey) [inline wiki/Tournaments/TWC/2023/en.md:183]: in what way could you purposefully disregard this... ? i don't think i understand this rule

File: wiki/Tournaments/TWC/2023/en.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+10. The tiebreaker will be played under Free Mod conditions, but players will be exempt from the mod requirement.
```
Comment (Walavouchey) [inline wiki/Tournaments/TWC/2023/en.md:184]: consistency

```suggestion
10. The tiebreaker will be played under FreeMod conditions, but players will be exempt from the mod requirement.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
good thing the rules are similar to OWC 2022, so i'm able to quickly review the changed parts

File: wiki/Tournaments/TWC/2023/en.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+| Round of 32 | 2023-03-25/2023-03-26 |
+| Round of 16 | 2023-04-01/2023-04-02 |
+| Quarterfinals | 2023-04-08/2023-04-09 |
+| Semifinals | 2023-04-15/2023-04-16 |
+| Finals | 2023-04-22/2023-04-23 |
+| Grand Finals | 2023-04-29/2023-04-30 |
+
+## Prizes
+
+The osu!taiko World Cup 2023 offers a minimum $2,000 cash prize pool and limited edition merch. The prize pool can be further increased from community donations.
```
Comment (Walavouchey) [inline wiki/Tournaments/TWC/2023/en.md:30]:
```suggestion
The osu!taiko World Cup 2023 offers a minimum $2,000 cash prize pool and limited-edition merch. The prize pool can be further increased from community donations.
```

**Comment by Walavouchey** [comment]
> PRing this ahead of time to split it from the (upcoming) news article PR.

are you intending to get the article in before the news post or at the same time?

**Comment by LeoFLT** [comment]
> > PRing this ahead of time to split it from the (upcoming) news article PR.
> 
> are you intending to get the article in before the news post or at the same time?

Article before the news post. That one should be PRd tomorrow or Thursday

**Comment by LeoFLT** [comment]
Everything has been addressed (I think)

**Review by Walavouchey — DISMISSED** [review]

File: wiki/Tournaments/en.md [hunk 4]
```diff
@@ -44,6 +44,7 @@ The osu! World Cup is a country-based 4v4 Team VS tournament following double-el
 | [osu!taiko World Cup 2020](TWC/2020) | 2020-03-07 | 2020-04-12 | ::{ flag=JP }:: Japan | ::{ flag=KR }:: South Korea | ::{ flag=GB }:: United Kingdom |
 | [osu!taiko World Cup 2021](TWC/2021) | 2021-03-06 | 2021-04-11 | ::{ flag=JP }:: Japan | ::{ flag=KR }:: South Korea | ::{ flag=BR }:: Brazil |
 | [osu!taiko World Cup 2022](TWC/2022) | 2022-03-12 | 2022-04-24 | ::{ flag=JP }:: Japan | ::{ flag=KR }:: South Korea | ::{ flag=ID }:: Indonesia |
+| [osu!taiko World Cup 2023](TWC/2023) | 2023-03-16 | 2023-04-30 | *TBD* | *TBD* | *TBD* |
```
Comment (Walavouchey) [inline wiki/Tournaments/en.md:47]: did you mean 2023-03-12 for qualifiers? 2023-03-16 does not correspond to any date on the tournament schedule

Comment (LeoFLT) [reply wiki/Tournaments/en.md:47]: 2023-03-18 for the Qualifier stage, 03-16 is a typo (02-16), but that date is also inconsistent with the current dates

**Review by Walavouchey — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**